firmware/papracode/LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
It doesn't look like we have a license file in the papra-pcb-firmware repo, if that isn't on purpose can we make an issue in that repo?

Reply:
Firmware is in a submodule, I'll create the license file issue over there.